examples/contour/02-job-certgen.yaml

```diff
@@ -5,6 +5,78 @@ metadata:
   name: contour-certgen
   namespace: projectcontour
 ---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: contour-certgen
```

Comment:
Because the Contour pods will also end up with this PSP, I'd probably recommend fixing the name to be `contour-psp` or something.